src/pip/_internal/baseparser.py

```diff
@@ -235,6 +241,22 @@ def get_default_values(self):
                 defaults[option.dest] = option.check_value(opt_str, default)
         return optparse.Values(defaults)
 
+    def invalid_config_error_message(self, action, key, val):
+        """Returns a better error message when invalid configuration option
+        is provided."""
+        if action in ('store_true', 'store_false'):
+            error_message = "{0} is not a valid value for "\
```

Comment:
You can use parenthesis here.

```
return (
 "..."
 "..."
).format(...)
```

(on mobile, excuse the lack of brevity)

Reply:
Done.